build/persistence/postgres/Dockerfile

```diff
@@ -0,0 +1,32 @@
+###############################################################################
+# (C) Copyright IBM Corp. 2020
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################################
+
+FROM postgres:12.3-alpine
+
+# Hard Coded environment values
+ENV POSTGRES_DB fhirdb
+ENV POSTGRES_USER postgres
+ENV PGDATA /db/data
+ENV LANG en_US.utf8
+
+# Create a working data directory
+RUN mkdir -p /db/data && chown -R postgres:postgres /db
+VOLUME /db/data
+WORKDIR /db
+
+# Open up the port
+EXPOSE 5432
+
+# Load Build Information
+COPY --chown=postgres:postgres resources/db.sql /docker-entrypoint-initdb.d/db.sql
+COPY --chown=postgres:postgres resources/docker-entrypoint.sh /docker-entrypoint.sh
+RUN chmod 755 /docker-entrypoint.sh
+
+# Add the fhiradmin user
+RUN addgroup -S fhir && adduser -S fhiradmin -G fhir
+RUN echo "fhiradmin:fhir" | chpasswd
```

Comment:
does this set a password?

Reply:
yeah - to fhir, I'll fix this

Reply:
So it uses chpasswd that's available in alpine examples... 
https://linoxide.com/linux-command/change-passwords-batch-mode-chpasswd/

Reply:
I actually removed this.